src/visualization/efmviz/mapExpressionToReactions_efmviz.m

```diff
@@ -0,0 +1,38 @@
+function [expressionRxns, parsedGPR] = mapExpressionToReactions_efmviz(model, expressionData, minSum)                                          
```

Comment:
Do we need the duplication of this file for EFMViz? Wouldn't it be better to add changes to the original file?